vars/main.yml

```diff
@@ -1,7 +1,4 @@
 ---
-host_name: localhost
-port: 9300
-rule_dir: rules
```

Comment:
Why we have removed this?

Reply:
Because of https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#understanding-variable-precedence
In current version I can't set these variables in playbook if they are defined on role level (instead of using `defaults` for that case).